engines/director/detection_tables.h

```diff
@@ -4952,6 +5019,12 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1_l("wbz", "", "WBZ.EXE", "38b75ecdedf662326fe4931a68ae60cd", 1410284, Common::JA_JPN, 501),
 
+	// Original filename is 'カードバトル'
+	MACGAME1_l("wrestlelife", "", "xn--lckycc2a0k0d", "c08abfb9c78cf67386f12b92a7481422", 718728, Common::JA_JPN, 501),
+	// Decomposed variant of above filename
+	MACGAME1_l("wrestlelife", "", "xn--2bka6c5fc4b3n0e", "c08abfb9c78cf67386f12b92a7481422", 718728, Common::JA_JPN, 501),
+	WINGAME1t_l("wrestlelife", "", "BATTLE.EXE", "4a212b51566fdde40f0abcd811d335cc", 1394459, Common::JA_JPN, 505),
```

Comment:
Same as my comment below on 6.0.6.

The windows binaries contain strings like `5.0.5.0r134`. This is really the major and minor versions smushing together. It should be read as `5.0 / 5.0r134`.

In that example, the string that I used came from this file, as Windows reads the file header:

![image](https://user-images.githubusercontent.com/54326852/151718391-3bc3caca-da99-4bab-8eda-d3d170f17946.png)

Reply:
Oops, good to know! Fixed.